### PR TITLE
Housekeeping: Dependency update 04-2026

### PR DESCRIPTION
**Pull request by KCeh**
## Summary

Just dependency update

**Related issue**: 

## Changes

- Bump in AGP
- Bump in other dependencies
- Fix dokka config

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [x] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [ ] I have updated the documentation to reflect my changes (if applicable).

### CODE REVIEW

File: dbinspector-no-op/build.gradle [hunk 1]
```diff
@@ -2,12 +2,10 @@ import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 
 plugins {
     id "com.android.library"
```
[the remark this answers is not in the document]
Comment (KCeh) [reply dbinspector-no-op/build.gradle:4]: This is not needed for AGP 9.0+

File: sample/build.gradle [hunk 2]
```diff
@@ -2,12 +2,10 @@ import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 
 plugins {
     id "com.android.application"
```
[the remark this answers is not in the document]
Comment (KCeh) [reply sample/build.gradle:4]: This is not needed for AGP 9.0+

**Comment by KCeh** [comment]
We have flaky tests, I will just merge this
